tests/test_channel_metrics/test_completely_bounded_trace_norm.py

```diff
@@ -0,0 +1,32 @@
+"""Tests for completely_bounded_trace_norm"""
+import numpy as np
+
+from toqito.channels.dephasing import dephasing
+from toqito.channel_metrics.completely_bounded_trace_norm import completely_bounded_trace_norm
+
+
+def test_cb_trace_norm_quantum_channel():
+    """The diamond norm of a quantum channel is 1"""
+    phi = dephasing(2)
+    np.testing.assert_equal(completely_bounded_trace_norm(phi), 1)
+
+
+def test_cb_trace_norm_unitaries_channel():
+    """The diamond norm of phi = id- U id U* is the diameter of the smallest circle that contains the eigenvalues of U"""
+    U = 1/np.sqrt(2) * np.array([[1, 1], [-1, 1]]) # Hadamard gate
+    phi = np.block([[np.eye(2), np.eye(2)],[U, -U]])
```

Comment:
If I may jump in: 
The example in [QETLAB](https://qetlab.com/DiamondNorm#A_difference_of_unitaries_channel) specifies the channel as a cell of left and right Kraus operators. Your implementation of `completely_bounded_trace_norm` works only for Choi inputs. This means you'll first need to obtain a Choi representation of the channel (which is not as simple as stacking the matrices together).

`toqito` already provides this functionality:
```suggestion
    phi = kraus_to_choi([[np.eye(2), np.eye(2)],[U, -U]])
```
and of coure you have to import it first.
```python
from toqito.channel_ops import kraus_to_choi
```

Making this change will let you replace `1/4` factor in https://github.com/vprusso/toqito/blob/c39ac220126d8094c3ed356af1b2edd4b1605cca/toqito/channel_metrics/completely_bounded_trace_norm.py#L63 with `1/2` which is what QETLAB uses and it's in agreement with the SDP formulation found in literature.

Reply:
amazing @georgios-ts, thanks so much for solving it! I was not at all sure why it needs this extra 1/2 resulting in a 1/4, but great that it is not at all the case, that makes me understand it.

Reply:
Outstanding, and thank you, @georgios-ts for the clear and concise solution, and thank you @AmanieOxana for making the change!

Reply:
@vprusso do you have reviewed the code? Do you have further ideas?